src/tests/FunctionalTests/Android/Device_Emulator/StartupHook/Android.Device_Emulator.StartupHook.Test.csproj

```diff
@@ -14,6 +14,8 @@
   </ItemGroup>
 
   <ItemGroup>
-      <ProjectReference Include="..\..\..\TestAssets\StartupHookForFunctionalTest\StartupHookForFunctionalTest.csproj" />
+      <ProjectReference Include="..\..\..\TestAssets\StartupHookForFunctionalTest\StartupHookForFunctionalTest.csproj">
```

Comment:
This doesn't seem trivial to get working at the moment, should we just disable this test so we can atleast have coverage for all other tests?